pset-functions.js

```diff
@@ -207,6 +328,27 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 5:')
 
 // Add your code below this line
+let problem5test = "apple"
+console.log(`Test Word: ${problem5test}`)
+
+function doubleLetters(word){
+  arrwords = word.split("")
+  testcase = arrwords[0]
+
+  for(let i = 1; i<arrwords.length; i++){
+    if (testcase === arrwords[i]){
+      return true;
+    }
+    else {
```

Comment:
@kertcodes no need for an else statement here, you can get away with just having the `if`

```
if (testcase === arrwords[i]) {
  return true
}
```